main_stylesheet.css

```diff
@@ -0,0 +1,127 @@
+
+* {
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+}
+
+a:link, a:visited {
+    color: #CE3B45;
+}
+
+a:hover {
+    color: #215097;
+}
+
+html {
+    font-size: 16px;
+    font-family: "Times New Roman";
+}
+body{
+    margin: 0px;
+    padding: 0px;
+}
+
+Header {
+    padding: 60px;
+    text-align: center;
+    background: #6B3488;
+    display: flex;
+    flex-direction: column-reverse;
+    margin-bottom: 25px;
+
+}
+
+h3{
+    margin: 5px;
+}
+
+h2{
+    margin: 5px;
+    padding-bottom: 10px;
+}
+
+.mainHeading{
+    padding:15px;
+    color: white;
+    font-size: 4rem;
+    font-family: gothic;
+}
+.box {
+    border: solid;
+    background-color: #D3D3D3;
+    margin: 15px;
+    flex-grow:4;
+    display: flex;
+    flex-direction: column;
+    justify-items: left;
+
+}
+
+.flexContainer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
```

Comment:
- when resizing down to around mobile width the child containers outgrow the parent cycles box. I think the fix might be overflow: auto might be the fix (i hate flex, not the best person to diagnose) https://stackoverflow.com/questions/41674979/flex-child-is-growing-out-of-parent

Reply:
fixed by turning of flex in mobile view